app/konnect-platform/cmek.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Customer-Managed Encryption Keys (CMEK)
+content_type: reference
+layout: reference
+description: 'Use Customer-Managed Encryption Keys (CMEK) in {{site.konnect_short_name}} to encrypt sensitive data using keys from your AWS Key Management Service (KMS) account.'
+breadcrumbs:
+  - /konnect/
+
+
+products:
+    - konnect-platform
+tags:
+  - encryption
+  - security
+  - compliance
+works_on:
+  - konnect
+faqs:
+  - q: Can I use automation to configure CMEK in {{site.konnect_short_name}}?
+    a: No, CMEK can currently only be configured via the {{site.konnect_short_name}} UI. Once CMEK is configured, {{site.konnect_short_name}} data that is encrypted using CMEK will be visible from the {{site.konnect_short_name}} APIs, decK, and Terraform after decryption.
```

Comment:
The answer to this has changed. We are releasing UI, API and Terraform together. Customers will be able to use UI, API and Terraform to configure CMEK.